src/popsift/popsift.h

```diff
@@ -82,6 +82,11 @@ class PopSift
         popsift::ImageBase*                    _current;
 
         popsift::Pyramid*                      _pyramid;
```

Comment:
Would be good to initialize all pointers.
`popsift::Pyramid*                      _pyramid = nullptr;`

Reply:
I think that _current is actually a leftover (deprecated). It was replaced by SiftJob to handle both uchar and float input images, and I forgot to remove it.

Reply:
yes i forgot to remove it...

Reply:
done in 82296d7  and e2f56ba